tools/hive/kerberos/marathon/hdfs-hive-kerberos.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
In other tests we generate these kinds of application definitions on the fly in python -- templating where applicable. Not a blocker, but we could create a follow-up ticket.